UEBPrep/app/assets/javascripts/cards.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function() {
+    $('#card_type').change(function(){
+        $('#card_title_tr').remove();
+        $('#card_textarea_tr').remove();
```

Comment:
for sure :)  there's some other js stuff i need to update/remove also.. like i don't think we even need my external jquery script at all - i'm pretty sure i noticed we installed its gem. also brandon and i are implementing zurb friday which will also just change everything all over again, so we'll just play it by ear !